.github/workflows/stale.yml

```diff
@@ -57,14 +59,20 @@ jobs:
 
           # Labels on issues exempted from stale.
           exempt-issue-labels: |
-            "Status: Blocked,Status: Decision Required,Peloton 🚴‍️"
+            "Status: Blocked,Status: Decision Required,Peloton 🚴‍♂️"
 
           # Labels on prs exempted from stale.
           exempt-pr-labels: |
-            "Status: Blocked,Status: Decision Required,Peloton 🚴‍"
+            "Status: Blocked,Status: Decision Required,Peloton 🚴‍♂️"
+
+          # Max number of operations per run.
+          operations-per-run: 30
```

Comment:
Let's make this explicit, even though it's the default value, as it may be something that we want to change given the observed behaviour of the GHA